src/pages/home/owners/InvitationsTable.tsx

```diff
@@ -0,0 +1,199 @@
+        <TableBody>
+          { invites.map(
+            ({
+              collaborationId,
+              coverArtUrl,
+              duration,
+              firstName,
+              lastName,
+              pictureUrl,
+              role,
+              royaltyRate,
+              status,
+              title,
+            }) => (
+              <TableRow key={ collaborationId }>
+                <TableCell>
+                  <Stack
+                    sx={ {
+                      flexDirection: "row",
+                      alignItems: "center",
+                      columnGap: 1.5,
+                      whiteSpace: "nowrap",
+                    } }
+                  >
+                    { coverArtUrl ? (
+                      <img
+                        style={ {
+                          borderRadius: "4px",
+                          width: "40px",
+                          height: "40px",
+                        } }
+                        src={ getResizedAlbumCoverImageUrl(coverArtUrl, {
+                          width: 50,
+                          height: 50,
+                        }) }
+                        alt="Song cover"
+                      />
+                    ) : (
+                      <Stack sx={ { height: "40px", width: "40px" } }></Stack>
+                    ) }
+                    { title }
+                  </Stack>
+                </TableCell>
+                <TableCell>
+                  <Stack
+                    sx={ {
+                      flexDirection: "row",
+                      alignItems: "center",
+                      columnGap: 1.5,
+                      whiteSpace: "nowrap",
+                    } }
+                  >
+                    { pictureUrl ? (
+                      <img
+                        style={ {
+                          borderRadius: "50%",
+                          width: "40px",
+                          height: "40px",
+                        } }
+                        src={ pictureUrl }
+                        alt="Profile"
+                      />
+                    ) : (
+                      <Stack sx={ { height: "40px", width: "40px" } }></Stack>
+                    ) }
+                    { firstName && lastName ? `${firstName} ${lastName}` : null }
+                  </Stack>
+                </TableCell>
+                <TableCell>{ role }</TableCell>
+                <TableCell sx={ { textAlign: "end" } }>
+                  { duration
+                    ? convertMillisecondsToSongFormat(duration)
+                    : "--:--" }
+                </TableCell>
+                <TableCell>{ `${royaltyRate}%` }</TableCell>
+                <TableCell>
+                  { status === "Editing" ? (
+                    <Stack
+                      flexDirection="row"
+                      columnGap={ 2 }
+                      justifyContent="end"
+                    >
+                      <IconButton
```

Comment:
Does our `Button width="icon"` work well with the MUI icons or does it work better with their preset `IconButton`? Should that be a post-MVP type refactor?

Reply:
It does, but it doesnt play well with our colors, i didnt want to mix them up. Could be a post mvp task bc i feel like ux is still not set on what the variations of button and icon buttons should be. dont want to open that can of worms just yet